### PR TITLE
Add Douban FM.app version 1.0

### PR DESCRIPTION
**Pull request by vinkla**
DoubanFM Desktop Client Powered by Atom-shell https://github.com/cyrilis/Douban-FM-Express

### CODE REVIEW

File: Casks/douban-fm.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'douban-fm' do
+  version '1.0'
+  sha256 'cbd00a0e035004feabf74181730d6a9ee3840da11ecf0e5e5d53094a26663456'
+
+  url "https://github.com/cyrilis/Douban-FM-Express/releases/download/v#{version}/Douban.FM.app.#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/douban-fm.rb]: Could we have a `appcast 'https://github.com/cyrilis/Douban-FM-Express/releases.atom'` line under this?

**Comment by cyrilis** [comment]
Yes, thanks,  go ahead.
I'm glad you like it. :smile:
then I could just update my release page to keep it updated in castroom, right?

File: Casks/douban-fm.rb (hunk 1 above)
Comment (vinkla) [inline Casks/douban-fm.rb]: Done.

File: Casks/douban-fm.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'douban-fm' do
+  version '1.0'
+  sha256 'cbd00a0e035004feabf74181730d6a9ee3840da11ecf0e5e5d53094a26663456'
+
+  appcast 'https://github.com/cyrilis/Douban-FM-Express/releases.atom'
```
Comment (vitorgalvao) [inline Casks/douban-fm.rb]: Should be under `url`, not above.

**Comment by vitorgalvao** [comment]
> I could just update my release page to keep it updated in caskroom, right?

Not yet, but in the future (feature isn’t done, yet).

**Comment by cyrilis** [comment]
@vitorgalvao  Got it. Thanks.

**Pull request merged**